packages/frontend/src/components/Button/style.module.scss

```diff
@@ -35,15 +35,6 @@
     }
   }
 
-  &.secondary {
-    background-color: var(--buttonSecondaryBackground);
-    color: var(--buttonSecondaryText);
-    border: 1px solid #ccc;
-    &:hover {
-      background-color: var(--buttonSecondaryHover);
```

Comment:
Just FYI, hover disabled primary buttons look a little weird, but this is not about this MR.

![image](https://github.com/user-attachments/assets/fb2f14f4-6368-4d29-b5af-3af143937cd0)

Reply:
is fixed